<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="SOLNYU-YGG9ACR1" name="net.microscraper.test.ScraperTest" tests="1" failures="1" timestamp="1 Jun 2011 20:03:03 GMT" time="0.111" errors="0">
  <testcase name="testSimpleGoogleScraper" time="0.111" classname="net.microscraper.test.ScraperTest">
    <failure type="java.lang.ExceptionInInitializerError">
      <![CDATA[java.lang.ExceptionInInitializerError
	at net.microscraper.test.ScraperTest.testSimpleGoogleScraper(ScraperTest.java:70)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:81)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:673)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:842)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1166)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1150)
	at org.testng.TestRunner.privateRun(TestRunner.java:752)
	at org.testng.TestRunner.run(TestRunner.java:603)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:329)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:324)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:286)
	at org.testng.SuiteRunner.run(SuiteRunner.java:235)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1049)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:974)
	at org.testng.TestNG.run(TestNG.java:905)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:203)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:174)
Caused by: java.lang.IllegalStateException: Native library for Attach API not available in this JRE
	at mockit.internal.startup.JDK6AgentLoader.getVirtualMachineImplementationFromEmbeddedOnes(JDK6AgentLoader.java:81)
	at mockit.internal.startup.JDK6AgentLoader.loadAgent(JDK6AgentLoader.java:54)
	at mockit.internal.startup.AgentInitialization.initializeAccordingToJDKVersion(AgentInitialization.java:21)
	at mockit.internal.startup.Startup.verifyInitialization(Startup.java:196)
	at mockit.Invocations.<clinit>(Invocations.java:25)
	... 26 more
Caused by: java.lang.UnsatisfiedLinkError: no attach in java.library.path
	at java.lang.ClassLoader.loadLibrary(Unknown Source)
	at java.lang.Runtime.loadLibrary0(Unknown Source)
	at java.lang.System.loadLibrary(Unknown Source)
	at sun.tools.attach.WindowsVirtualMachine.<clinit>(WindowsVirtualMachine.java:169)
	at mockit.internal.startup.JDK6AgentLoader.getVirtualMachineImplementationFromEmbeddedOnes(JDK6AgentLoader.java:68)
	... 30 more
]]>
    </failure>
  </testcase>
</testsuite>
